source/manage-apps/configure/config.txt

```diff
@@ -38,6 +38,14 @@ using structured JSON configuration and JavaScript source code files.
 - ``.js`` files define serverless application logic used in functions, triggers,
   HTTPS endpoints, and custom resolvers.
 
+.. important:: Reserved File Names
+   
+   You can include arbitrarily named files in your realm app config, with a few exceptions.
+   The following file names should not be used: ``stitch.json``, ``config.json``, 
+   ``realm_config.json``, and ``app_config.json``. Files with these names
+   conflict with pre-made files of the same name in their app structure's file
```

Comment:
well it's not that they shouldn't be used, it's that these are reserved file names, so they should be used only for these reserved scenarios. 

and the files aren't only 'pre-made'. you can make them following the structure specified in the app configuration reference - https://www.mongodb.com/docs/atlas/app-services/manage-apps/configure/config/